test/test_test.c

```diff
@@ -531,6 +531,16 @@ static int test_bn_output(int n)
     return 1;
 }
 
+static int test_skip_one(void)
+{
+    return TEST_skip("skip test");
+}
+
+static int test_skip_many(int n)
+{
+    return TEST_skip("skip many test: %d", n);
+}
+
```

Comment:
Should there be a test with a NULL argument?

Reply:
A good question.  Does skipping a test without giving a reason make sense?
The current code accepts a NULL, so somebody will try it.

Reply:
A test has been added.